From ef031a57e64e74381abcdb424f08182bc81600a1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Fri, 10 Sep 2010 17:43:13 +0000
Subject: [PATCH 114/160] gettext docs: add a "Testing your changes" section to po/README
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change po/README to tell translators that they should do `make` at the
top-level before submitting their translations. Many common errors
will be caught with the `msgfmt --check` run that the Makefile will
compile the message catalogs with.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 po/README |   13 +++++++++++++
 1 files changed, 13 insertions(+), 0 deletions(-)

diff --git a/po/README b/po/README
index 51d6f3b..8ba06c8 100644
--- a/po/README
+++ b/po/README
@@ -56,3 +56,16 @@ and then in the po/ directory do:
     msgmerge --add-location --backup=off -U XX.po git.pot
 
 Where XX.po is the file you want to update.
+
+Testing your changes
+--------------------
+
+Before you submit your changes go back to the top-level and do:
+
+    make
+
+On systems with GNU gettext (i.e. not Solaris) this will compile your
+changed PO file with `msgfmt --check`, the --check option flags many
+common errors, e.g. missing printf format strings, or translated
+messages that deviate from the originals in whether they begin/end
+with a newline or not.
-- 
1.7.3.1.50.g1e633

